namespaces/Representor.md

```diff
@@ -0,0 +1,133 @@
+# Representor Namespace
+
+This document extends [Refract](../refract-spec.md) Specification to describe the [Representor](https://github.com/the-hypermedia-project/charter) DOM.
+
+## Representor (Object Element)
+
+The Representor represents a Hypermedia Resource, with it's available links, transitions and attributes.
+
+### Properties
+
+- `links` - (Array Element of Link) - The links from this resource.
```

Comment:
I think you want to follow Refract here. So instead top-level properties, nest this under either `content` or `attributes`. The way you have written it here does not follow refract concepts where everything is an element of `element`, `meta`, `attributes` and `content` properties.